### PR TITLE
Use "function" instead of "method" where appropriate

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

File: en/django_views/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ def post_list(request):
 ```
 
 
-As you can see, we created a method (`def`) called `post_list` that takes `request` and `return` a method `render` that will render (put together) our template `blog/post_list.html`.
+As you can see, we created a function (`def`) called `post_list` that takes `request` and `return` a function `render` that will render (put together) our template `blog/post_list.html`.
```
Comment (aniav) [inline en/django_views/README.md]: `post_list` is a method. No need to change that ;)

Comment (domenkozar) [inline en/django_views/README.md]: In Python terms, it's not a method: https://docs.python.org/2/tutorial/classes.html#method-objects

Comment (domenkozar) [inline en/django_views/README.md]: Correct me if I'm wrong, if it's a method, to what class does it belong?

Comment (aniav) [inline en/django_views/README.md]: Ah, sorrry! I messed this in my head with the class based views. You are totally right, thank you for correcting me!

Comment (domenkozar) [inline en/django_views/README.md]: :cake:

**Pull request merged**